apps/recovery-portal/src/main/resources/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/Resources.properties

```diff
@@ -58,6 +58,7 @@ Password.cannot.be.empty=Password cannot be empty.
 Updated.the.password.successfully=Updated the password successfully
 Server.failed.to.respond=Server failed to respond.
 No.valid.user.found=No valid user found
+Insufficient.info.to.find.user=Provided information are insufficient to identify the user
```

Comment:
```suggestion
Insufficient.info.to.find.user=Provided information is insufficient to identify the user
```

Reply:
Addressed in https://github.com/wso2/identity-apps/pull/1245/commits/a2b9c00bf6d0fb5c25efcf61feff8db6e2b45cff